sdk/azidentity/chained_token_credential_test.go

```diff
@@ -153,6 +153,220 @@ func TestChainedTokenCredential_GetTokenWithUnavailableCredentialInChain(t *test
 	}
 }
 
+func TestChainedTokenCredential_ChecksThatSuccessfulCredentialIsSet(t *testing.T) {
```

Comment:
Does this test anything the case below does not?

Reply:
No, just trying to have smaller tests. I’ll remove this one.